civitai-downloader-v2/src/core/download/manager.py

```
@@ -0,0 +1,670 @@
+"""
+Download Manager for CivitAI Downloader.
+Provides concurrent downloads with resume capability, progress tracking, and integrity verification.
+"""
+
+import os
+import asyncio
+import aiohttp
+import hashlib
+import time
+from pathlib import Path
+from dataclasses import dataclass, field
```

Ruff (0.12.2)

`dataclasses.field` imported but unused

Remove unused import: `dataclasses.field`

(F401)